example/config/deploy.rb

```diff
@@ -4,3 +4,4 @@
 set :application, 'my_app_name'
 set :repo_url, 'git@github.com:MindscapeHQ/capistrano-raygun-deployment.git'
 set :deploy_to, '/var/www/my_app_name'
+set :raygun_deployment_environments, [:production]
```

Comment:
Cool, I can adopt any way, Just trying to make sure it will be no problem when some one doesn't pay attention for that hehe but Probably not. Thanks. I think It's good for merging.